spec/models/adventure_spec.rb

```diff
@@ -3,27 +3,19 @@
 describe Adventure do
   subject { FactoryGirl.build(:adventure) }
 
-  it { should have_many(:bucket_list_adventures) }
+  it { should have_many(:bucket_list_adventures).dependent(:destroy) }
+  it { should have_many(:users) }
 
-  # it { should have_valid(:name).when('Visit Sue', 'Eat alligator') }
-  # it { should have_valid(:address).when('Paris, France', '33 Harrison Ave, Boston') }
-  # it { should_not have_valid(:name).when(nil, '') }
-  # it { should_not have_valid(:address).when(nil, '') }
+  it { should have_valid(:name).when('Visit Sue', 'Eat alligator') }
+  it { should have_valid(:address).when('Paris, France', '33 Harrison Ave, Boston') }
 
-  it {should allow_value(nil).for(:latitude)}
-  # it { should have_valid(:latitude).when(40.7127837, 39.904211) }
-  # it { should_not have_valid(:latitude).when('cat', '.') }
-  # it { should validate_numericality_of(:latitude).is_greater_than_or_equal_to (-180) }
-  # it { should validate_numericality_of(:latitude).is_less_than_or_equal_to (180) }
+  it { should have_valid(:latitude).when(40.7127837, 39.904211) }
+  it { should_not have_valid(:latitude).when('cat', '.') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.